src/Resources/config/services/conditional/cache.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+
+        <service id="setono_sylius_shipping_countdown.provider.next_shipment.cached"
+                 class="Setono\SyliusShippingCountdownPlugin\Provider\CachedNextShipmentDateTimeProvider"
+                 decorates="setono_sylius_shipping_countdown.provider.next_shipment">
```

Comment:
I guess you are missing a decoration priority

Reply:
Done ;)